src/Datadog.Trace/Configuration/IntegrationIds.cs

```diff
@@ -34,5 +34,6 @@ internal enum IntegrationIds
         CosmosDb,
         AwsSdk,
         AwsSqs,
+        AzureFunction,
```

Comment:
I think this should be `AzureFunctions` to be consistent with the product's branding.